src/Compiler/Lexer.cpp

```diff
@@ -162,10 +162,25 @@ namespace Ark::internal
                         {
                             switch (ctrl_char[0])
                             {
-                                /// @todo
-                                case 'x': break;
-                                case 'u': break;
-                                case 'U': break;
+                                case 'x': break; /// @todo
+
+                                case 'u':
+                                {
+                                    char *utf8_str = utf8decode(ctrl_char.c_str() + 1);
+                                    buffer += utf8_str;
+                                    free(utf8_str);
+                                    break;
+                                }
+
+                                case 'U':
+                                {
+                                    short begin = 1;
+                                    for(; ctrl_char[begin] == '0'; ++ begin);
+                                    char *utf8_str = utf8decode(ctrl_char.c_str() + begin);
```

Comment:
for coding style:
```cpp
for (; ctrl_char[begin] == '0'; ++begin);
char* utf8_str = utf8decode(ctrl_char.c_str() + begin);
```

Reply:
i guess coding style is respected

Reply:
see the spaces and placement of * on the type

Reply:
Ok